samples/complex/dispatcher/dotnet/Dispatcher/Utils/Utilities.cs

```diff
@@ -0,0 +1,182 @@
+﻿using Microsoft.Agents.Authentication.Msal;
+using Microsoft.Agents.Authentication;
+using Microsoft.Identity.Client;
+using System.Reflection;
+using DispatcherAgent.Model;
+using System.Collections.Concurrent;
+using System.Diagnostics;
+using Microsoft.Agents.BotBuilder;
+using Microsoft.Agents.Core.Interfaces;
+using Microsoft.Agents.Core.Models;
+using Microsoft.Agents.Storage;
+
+namespace DispatcherAgent.Utils
+{
+    public static class Utilities
```

Comment:
Can we split the methods in this class to other classes with better names? Seems to me the methods to get the conversation links should be in a different class that then ones related to oauth flows